course/assets/course.js

```diff
@@ -13,6 +13,7 @@ import { initCourseInfoExpander } from "./js/course_expander"
 import { initVideoTranscriptTrack } from "./js/video_transcript_track"
 import { initPlayBackSpeedButton } from "./js/video_playback_speed"
 import { initVideoFullscreenToggle } from "./js/video_fullscreen_toggle"
+import { initDownloadButton } from "./js/video-download-button"
```

Comment:
Since `video-download-button.js` consist of the code of both download buttons: 1. Video download, 2. Transcript Download so maybe, either we can make separate files for that or rename this file (something like `video-transcript-download-buttons`) so it indicates/represents both.